uniref-rest/src/main/java/org/uniprot/api/uniref/repository/store/UniRefEntryStoreRepository.java

```diff
@@ -0,0 +1,151 @@
+package org.uniprot.api.uniref.repository.store;
```

Comment:
I liked the arrangement of methods. It helped me to read and understand the logic quickly. 👍